{"status":"TIMEOUT","subtests":[{"name":"Return NotSupportedError when calling observer()","status":"PASS"},{"name":"Basic functionality test","status":"TIMEOUT"},{"name":"Removing observer before observe() resolves works","status":"NOTRUN"},{"name":"Calling observe() multiple times works","status":"NOTRUN"},{"name":"Starting a new observer after an observer has started works","status":"NOTRUN"}]}